syntax = "proto3";

import "ftrace_event.proto";

message FtraceEventBundle {
  uint32 cpu = 1;
  repeated FtraceEvent event = 2;
}
